system/Database/BaseUtils.php

```diff
@@ -413,11 +413,9 @@ public function backup($params = [])
 	/**
 	 * Platform dependent version of the backup function.
 	 *
-	 * @param array|null $prefs
-	 *
 	 * @return mixed
 	 */
-	abstract public function _backup(array $prefs = null);
+	public abstract function _backup();
```

Comment:
While it might be "proper" to call this abstract, I would prefer not to in this case.

Reply:
@lonnieezell do you want to remove this `_backup()` method ?

Reply:
I reverted Baseutils class change.